controllers/timeSplits.ts

```diff
@@ -0,0 +1,81 @@
+import prisma from "../prisma/client";
+import asyncHandler from "express-async-handler";
+import { TimeSplit } from "@prisma/client";
+import { formatError } from "../library/errors";
+import { isContentOwner } from "../library/userHelper";
+
+const create_time_splits = asyncHandler(async (req, res, next) => {
+  const { contentId, contentType, timeSplits } = req.body;
```

Comment:
we may want to add some validation checks here
```suggestion
  const { contentId, contentType, timeSplits } = req.body;
  if (typeof contentId !== "string" && typeof contentType !== "string" && typeof timeSplits !== "array") {
      const error = formatError(
      400,
      "contentId must be a string, contentType must be a string, and timeSplits must be an array"
    );
    next(error);
```